src/core/dbus-manager.c

```diff
@@ -1451,6 +1451,27 @@ int verify_run_space_and_log(const char *message) {
         return 0;
 }
 
+static void log_reload_caller(sd_bus_message *message, Manager *manager) {
+        _cleanup_(sd_bus_creds_unrefp) sd_bus_creds *creds = NULL;
+        Unit *caller;
+        pid_t pid;
+
+        assert(message);
+        assert(manager);
+
+        if (sd_bus_query_sender_creds(message, SD_BUS_CREDS_PID, &creds) < 0)
```

Comment:
Maybe add SD_BUS_CREDS_AUGMENT|SD_BUS_CREDS_COMM here, so that you can show the process name.

i.e. do what bus_client_log() does in resolved. it's a very similar case.

(oh, and I'd show the UID too, I think)

Reply:
I won't bother with UID as it makes the logic to make it optional awkward, and I'm not sure it would be that useful, unit pid and comms should be enough